pkg/auth/authz_handler_test.go

```diff
@@ -202,6 +202,22 @@ func TestAuthz(t *testing.T) {
 		assert.Contains(t, err.Error(), expectedCode)
 	}
 
+	waitForServerToBecomeReady := func(timeout time.Duration) {
+		start := time.Now()
+		for {
+			passAccessReview(1)
```

Comment:
Can those mocks break the test? if there some race or something like that?

Reply:
no, thought of that